src/Libraries/CoreNodeModels/Input/DateTime.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Dynamo;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using ProtoCore.AST.AssociativeAST;
+
+namespace DSCoreNodesUI
+{
+    [NodeName("Date Time")]
+    [NodeDescription("Create a DateTime object by specifying a date and a time.")]
+    [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
+    [IsDesignScriptCompatible]
+    public class DateTime : BasicInteractive<System.DateTime>
+    {
+        public DateTime()
```

Comment:
@pboyer You are correct. This bit of UI is out of scope. @lillismith and I have discussed this. This UI is a first pass implementation until we implement a better date picker control. We needed to have SOMETHING, that wasn't the user constructing a date time object with a minimum of 5 nodes.